### ISSUE TITLE
Add data layer to VBA Regional office pages

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-cms#16311]
## Description
We need to ensure that we are able to capture data about the visits and interactions on Regional offices pages, whether automatically through the use of design components or addition of custom code. 

## Tasks
- [ ] Determine coverage by components
- [ ] Add custom code as needed
- [ ] Request the addition of Regional offices as a product to the existing VA.gov Content KPI DOMO Dashboard
- [ ] Request the addition of Medallis/Foresee

## Acceptance criteria
The following data can be collected from Regional office pages
- [ ] Number of pageviews (total and unique)
- [ ] Number of users (total and unique, new and returning)
- [ ] Click interactions by accordion
- [ ] Click interactions by button
- [ ] Click interactions by phone number (and from which accordion)
- [ ] Click interactions by link (and from which accordion)
- [ ] Scroll depth

**Comment by davidmpickett** [comment]
I think this may be blocked by #15155

**Comment by xiongjaneg** [comment]
Eli met with Wes Rowe and verified some actions were already automatically tracked and can be tested.

**Comment by davidmpickett** [comment]
I started a draft of a analytics request here: https://github.com/department-of-veterans-affairs/va.gov-team/issues/76780

**Comment by maxx1128** [comment]
@eselkin, can you confirm, from your conversation with Wes Rowe, which of the acceptance criteria are already checked off due to the automatic tracking? I believe the number of page views and users are already tracked, but I don't know how many click interactions are or if the scroll depth is included.

**Comment by maxx1128** [comment]
Update for end of 104: I got a good start on this ticket near the end of this sprint and opened up [a draft pull request for it.](https://github.com/department-of-veterans-affairs/content-build/pull/1930) However, I got delayed by an unexpected bug for mental health numbers that appeared in production. So, this will carry over to the next sprint.

**Comment by xiongjaneg** [comment]
@maxx1128 Would you check off the ACs you've completed? Your note sounds like it's all of them, but just in case...

**Comment by maxx1128** [comment]
> @maxx1128 Would you check off the ACs you've completed? Your note sounds like it's all of them, but just in case...
> 
> Did you need to add any custom code?

I just updated the AC. No custom code was needed.

**Comment by xiongjaneg** [comment]
Perfect. Thank you!
